testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1292,6 +1292,9 @@ def server_secret
   source = File.dirname(__FILE__) + '/../upload_files/ansible/playbooks/example_playbook2_orion_dummy/hosts'
   return_code = file_inject(target, source, dest + "hosts")
   raise 'File injection failed' unless return_code.zero?
+  source = File.dirname(__FILE__) + '/../upload_files/ansible/playbooks/example_playbook2_orion_dummy/example_file.txt'
```

Comment:
What about having this other structure?
```
.
└── ansible
    └── playbooks
        ├── orion_dummy
        │   ├── file.txt
        │   ├── hosts
        │   └── playbook_orion_dummy.yml
        └── ping
            └── playbook_ping.yml
```

Reply:
Yeah, It makes sense to simplify this a bit. In any case, I'm keeping `playbook_ping.yml` in the main `playbooks` as I don't want that one to be contained in a custom folder as the `orion_dummy` one.